index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="container">
+        <div id="template1">
+            <div class="toolbar">
+                <div class="bar">
+                    <div class="time">9:41</div>
+                    <div class="network">
+                        <img src="images/header/tower.png" alt="network_img">
+                    </div>
+                    <div class="wifi">
+                        <img src="images/header/Wifi.png" alt="wifi_logo">
+                    </div>
+                    <div class="battery">
+                        <img src="images/header/Battery.png" alt="battery">
+                    </div>
+                </div>
+                <div class="myLogo">
+                    <div class="logo">My Logo</div>
+                    <div class="Union">
+                        <img src="images/header/Union.png" alt="bar">
+                    </div>
+                </div>
+            </div>
+            <main>
+                <div class="bg_image">
+                    <img class="background" src="images/header/header.png" alt="background_img">
+                </div>
+                <div class="intro">
+                    <h2>I’m Praveen Glad to see you!</h2>
+                </div>
```

Comment:
- [ ] Kindly adjust the spacing between the title and content. please see the image below for reference.
- [ ] kindly make the title bolder as well to stick to the design
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/191655774-b82003f2-b2e8-4005-b77c-4707668e73e5.png)
### Your design
![your](https://user-images.githubusercontent.com/81727988/191655794-5051db01-b8ab-4994-941e-cbf471a76856.png)

Reply:
Okay I will do it.
Thanks for guiding me.🙏